lib/utils.js

```diff
@@ -2,7 +2,9 @@
 
 // helper for convenient testing
 exports.require = (name) => {
-    const importedModule = require(require.resolve(name, {paths: [process.cwd()]})); // Require module using the absolute path
+    const defaultPathsToResolve = require.resolve.paths(name);
+    const absoluteModulePath = require.resolve(name, {paths: [process.cwd(), ...defaultPathsToResolve]});
```

Comment:
Found a bug which was reproduced for user with yarn@1.22.22, plugin is not found in `process.cwd`, but found in default folders from `require.resolve.paths`. So I fixed it.

Reply:
Jfyi: after this one we would not need to create symlinks for testing-project/node_modules 🔥